src/doc/trpl/looping.md

```diff
@@ -54,7 +54,7 @@ The other kind of looping construct in Rust is the `while` loop. It looks like
 this:
 
 ```{rust}
-let mut x = 5; // mut x: u32
+let mut x = 5;        // mut x: i32
```

Comment:
I think that might be easiest, yeah. Thanks for your effort! (Feel free to "@huonw" me on the new PR :smile:)